src/components/NcModal/NcModal.vue

```diff
@@ -409,13 +409,23 @@ export default {
 			type: Boolean,
 			default: false,
 		},
+
 		/**
 		 * Enable swipe between slides
+		 * @deprecated Will be removed in next version - use `disableSwipe` instead
```

Comment:
Not blocking, but don't forget to replace it for `next`
https://github.com/nextcloud-libraries/nextcloud-vue/blob/28860a17bb5024ec2edd2d8b1f9f24ff03b891dc/src/components/NcDialog/NcDialog.vue#L212